Google.Api.Gax.Grpc/ClientStreamingBase.cs

```diff
@@ -0,0 +1,116 @@
+﻿/*
+ * Copyright 2021 Google Inc. All Rights Reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using Grpc.Core;
+using System;
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax.Grpc
+{
+    /// <summary>
+    /// Base class for the client-side streaming RPC methods.
+    /// </summary>
+    /// <typeparam name="TRequest">RPC request type</typeparam>
+    /// <typeparam name="TResponse">RPC response type</typeparam>
+    public abstract class ClientStreamingBase<TRequest, TResponse>
+    {
+        /// <summary>
+        /// The underlying gRPC client streaming call.
+        /// Warning: DO NOT USE <c>GrpcCall.RequestStream</c> at all if using
+        /// <see cref="TryWriteAsync(TRequest)"/>, <see cref="WriteAsync(TRequest)"/>,
+        /// <see cref="TryWriteAsync(TRequest, WriteOptions)"/> , or <see cref="WriteAsync(TRequest, WriteOptions)"/>.
+        /// Doing so will cause conflict with the write-buffer used within the <c>[Try]WriteAsync</c> methods.
+        /// </summary>
+        public virtual AsyncClientStreamingCall<TRequest, TResponse> GrpcCall
```

Comment:
Expression-bodied member here and below?

Reply:
👍